src/pages/developers/tutorials.js

```diff
@@ -220,11 +222,25 @@ const TutorialsPage = ({ data, pageContext }) => {
       published: tutorial.frontmatter.published,
     }))
 
+  const externalTutorials = foreignTutorials.map((tutorial) => ({
+    to: tutorial.url,
+    title: tutorial.title,
+    description: tutorial.description,
+    author: tutorial.author,
+    tags: tutorial.tags.map((tag) => tag.toLowerCase().trim()),
+    skill: tutorial.skillLevel,
+    timeToRead: tutorial.timeToRead,
+    published: tutorial.publishDate,
+  }))
+
```

Comment:
So I would rearrange a few things here.  We start in this component by declaring `allTutorials` which is pulled from the graph query. 
1. I would _rename_ this to `internalTutorials`, both at the bottom where the query is declared, and also renaming this declaration to `const internalTutorials = data.internalTutorials.nodes`
2. I would cut this `.filter()` (L206) method from this declaration, and use it below in a moment.
3. Leave the `.map()` function (L211), but add `lang: tutorial.frontmatter.lang,` so we store the language for filtering in a moment
4. Then at this location (L235, after declaring both `internalTutorials` and `externalTutorials`) we can bring them together:
```js
const allTutorials = [...internalTutorials, ...externalTutorials]
```
Then apply the `.filter()` method from above, and add a `.sort()` to sort by date (honestly, reverse date would probably be better so new tutorials are highlighted at top... @minimalsm  thoughts?).

You should then be able to just use the new `allTutorials` array the same way as before with the existing logic, reverting below changes.

Let me know if that makes sense and if you're able to try those changes, or you need a hand getting it over the line. 👍🏻

Reply:
Agreed! Yes to reverse sort to show new tutorials at the top.